operator/src/main/java/oracle/kubernetes/weblogic/domain/model/AuxiliaryImage.java

```diff
@@ -38,17 +37,19 @@
   @EnumClass(ImagePullPolicy.class)
   private String imagePullPolicy;
 
-  @Description("The command for this init container. Defaults to `cp -R $AUXILIARY_IMAGE_PATH/* "
-      + "$AUXILIARY_IMAGE_TARGET_PATH`. This is an advanced setting for customizing the container command for copying "
-      + "files from the container image to the auxiliary image emptyDir volume. Use the `$AUXILIARY_IMAGE_PATH` "
-      + "environment variable to reference the value configured in `spec.auxiliaryImageVolumes.mountPath`, which "
-      + "defaults to \"/auxiliary\". Use '$AUXILIARY_IMAGE_TARGET_PATH' to refer to the temporary directory created by "
-      + "the operator that resolves to the auxiliary image's internal emptyDir volume.")
-  private String command;
+  @Description("The source location of the WebLogic Deploy Tooling installation within the auxiliary image. "
```

Comment:
that will be copied to '/aux/weblogic-deploy' in the WebLogic Server container in all pods.

Reply:
Fixed.